### PR TITLE
Homepg

### PR DESCRIPTION
**Pull request by dazyang**
Created three components - header, pings, ping. 
Mapping through pings array in data object. Rendered three pings found in mapped array.

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 import Profile from './Profile'
+import Pings from './Pings'
 
 const App = () => (
   <div className='app'>
+    <Header />
+    <WaitIndicator />
     <ErrorMessage />
-    <Route exact path='/' component={() => <h1>Pings</h1>} />
+    <Pings />
```
Comment (harrison-symes) [inline client/components/App.jsx]: Keep this as a Route for path '/' so you only see this on the home route

File: client/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+const Header = () => {
+  return (
+    <div className='header'>
+      <h1>Pings</h1>
+    </div>
+  )
+}
```
Comment (harrison-symes) [inline client/components/Header.jsx]: You could add a buttons/links (Link tag) that redirects to the /new page and the /profile page (and back to '/' again)

File: client/components/App.jsx [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
     <Route exact path='/newping' component={NewPing} />
```
Comment (harrison-symes) [inline client/components/App.jsx]: change /newping to /new

**Review by harrison-symes — CHANGES_REQUESTED** [review]

File: client/components/App.jsx [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
+import Pings from './Pings'
 
 const App = () => (
   <div className='app'>
+    <Header />
+    <WaitIndicator />
     <ErrorMessage />
-    <Route exact path='/' component={() => <h1>Pings</h1>} />
-    <Route exact path='/newping' component={NewPing} />
+    <Route exact path= '/' component={Pings} />
```
Comment (don-smith) [inline client/components/App.jsx]: Remove space

File: client/components/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { connect } from 'react-redux'
```
Comment (don-smith) [inline client/components/Header.jsx]: Remove space padding

File: client/components/Header.jsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      </ul>
+    </div>
+  )
+}
+
+// function mapStateToProps(state) {
+//   return {
+//     isHeaderVisable: state.isHeaderVisable
+//   }
+// }
```
Comment (don-smith) [inline client/components/Header.jsx]: Remove unhelpful comments

File: client/components/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+const Header = () => {
+  return (
+    <div className='header'>
+      <Link to='/'><h1>Pings</h1></Link>
+      <ul>
+        <li><Link to='/new'><strong>Create new ping</strong></Link></li>
+
+        <li><Link to='/profile'><strong>My Profile</strong></Link></li>
+
```
Comment (don-smith) [inline client/components/Header.jsx]: Remove blank line

File: client/components/Ping.jsx [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { connect } from 'react-redux'
```
Comment (don-smith) [inline client/components/Ping.jsx]: Remove space padding

File: client/components/Ping.jsx [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+      
+      <div className='image'>
+        Image:{props.ping.imageUrl}
+      </div>
+      
+      <div className='photo'>
+        Recepient:{props.ping.recepientPhoto}
+      </div>
+      <br/>
```
Comment (don-smith) [inline client/components/Ping.jsx]: Remove `<br/>`. We can do this with styling.

File: client/components/Pings.jsx [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { connect } from 'react-redux'
```
Comment (don-smith) [inline client/components/Pings.jsx]: Remove space padding

File: client/components/Pings.jsx [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+import Ping from './Ping'
+
+const Pings = (props) => {
+  return (
+    <div className='pings'>
+      {props.pings.map((ping, i) => {
+        return <div key={i}> <Ping ping={ping}/> </div>
```
Comment (don-smith) [inline client/components/Pings.jsx]: Use the `ping.id` instead of `i`.

File: client/components/Pings.jsx [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+const mapStateToProps = (state) => {
+  return {
+    pings: state.pings
+  }
+}
+
+export default connect(mapStateToProps)(Pings)
+
+//create links to home page by clicking ping header
+//link to new and to profile
```
Comment (don-smith) [inline client/components/Pings.jsx]: Remove comments

**Review by don-smith — CHANGES_REQUESTED** [review]

**Pull request merged**